server/utils/courseParticipationUtils.ts

```diff
@@ -152,80 +153,64 @@ export default class CourseParticipationUtils {
-    const summaryListRows: Array<GovukFrontendSummaryListRowWithValue> = []
-
-    summaryListRows.push({
-      key: { text: 'Programme name' },
-      value: { text: courseParticipationWithName.name },
-    })
-
-    if (courseParticipationWithName.setting) {
-      const valueTextItems: Array<string> = []
-
-      if (courseParticipationWithName.setting.type) {
-        valueTextItems.push(StringUtils.properCase(courseParticipationWithName.setting.type))
-      }
+    return [
+      CourseParticipationUtils.summaryListRow('Programme name', [courseParticipationWithName.name]),
+      CourseParticipationUtils.summaryListRowSetting(courseParticipationWithName),
+      CourseParticipationUtils.summaryListRowOutcome(courseParticipationWithName),
+      CourseParticipationUtils.summaryListRow('Additional detail', [courseParticipationWithName.outcome.detail]),
+      CourseParticipationUtils.summaryListRow('Source of information', [courseParticipationWithName.source]),
+      CourseParticipationUtils.summaryListRow('Added by', [
+        courseParticipationWithName.addedBy,
+        DateUtils.govukFormattedFullDateString(courseParticipationWithName.createdAt),
+      ]),
+    ]
+  }
 
-      if (courseParticipationWithName.setting.location) {
-        valueTextItems.push(`${courseParticipationWithName.setting.location}`)
-      }
+  // eslint-disable-next-line @typescript-eslint/member-ordering
+  private static summaryListRow(
+    keyText: GovukFrontendSummaryListRowKey['text'],
+    valueTextItems: Array<string | undefined>,
+  ): GovukFrontendSummaryListRowWithValue {
+    return {
+      key: { text: keyText },
+      value: { text: valueTextItems.join(', ') || 'Not known' },
+    }
+  }
 
-      const valueText = valueTextItems.join(', ')
+  private static summaryListRowOutcome(
+    courseParticipationWithName: CourseParticipationWithName,
+  ): GovukFrontendSummaryListRowWithValue {
+    const valueTextItems: Array<string> = []
 
-      if (valueText) {
-        summaryListRows.push({
-          key: { text: 'Setting' },
-          value: { text: valueText },
-        })
-      }
+    if (courseParticipationWithName.outcome?.status) {
+      valueTextItems.push(StringUtils.properCase(courseParticipationWithName.outcome.status))
     }
 
-    if (courseParticipationWithName.outcome) {
-      if (courseParticipationWithName.outcome.status) {
-        let valueText = ''
-
-        valueText += StringUtils.properCase(courseParticipationWithName.outcome.status)
+    if (courseParticipationWithName.outcome?.yearStarted) {
+      valueTextItems.push(`Year started ${courseParticipationWithName.outcome.yearStarted}`)
+    }
 
-        if (courseParticipationWithName.outcome.yearStarted) {
-          valueText += ` - started ${courseParticipationWithName.outcome.yearStarted}`
-        }
+    if (courseParticipationWithName.outcome?.yearCompleted) {
+      valueTextItems.push(`Year complete ${courseParticipationWithName.outcome.yearCompleted}`)
```

Comment:
I know this is per the designs, but given the tense of `Year started` above, I think this should be `Year completed` for consistency

Reply:
Will leave this as-is after running it past Louise